### ISSUE TITLE
Add 'USE mysql' statement when applying 'user' changes

### ISSUE THREAD

**Issue by atulwy** [webyog/sqlyog-community#1848]
Original [issue 1848](https://code.google.com/p/sqlyog/issues/detail?id=1848) created by webyog on 2012-08-29T10:39:57.000Z:

One user has reported that replication breaks when applying user changes from 'User manager' if server is using &quot;binlog-ignore-db&quot; option in my.cnf/ini.

&quot;SQLyog user manager does not set the current (default) database to &quot;mysql&quot; when applying user changes. This causes mysql to NOT write the change to the bin log and thus the changes are not replicate. This occurs when the servers are using the &quot;binlog ignore-db&quot; option in my.cnf.

In other words, just add &quot;USE mysql&quot; in front of all commands referring to users and privs.

This is just a work around. The real issue is the implementation of &quot;GRANT&quot; and like commands within MySQL server. The server needs to handle these hard user/priv commands correctly.&quot;

Keep an eye on this MySQL bug report: http://bugs.mysql.com/bug.php?id=66277
Basically GRANT statements does not address to any specific table or database. So without 'USE mysql' it shouldn't break replication.

Refer IB: 119556
